### PR TITLE
Three new lyrics

### PR DESCRIPTION
**Pull request by egecue**
- [Little Simz] add Venom from GREY Area
- [JID] add NEVER from The Never Story
- [Anderson .Paak] add Tints (feat. Kendrick Lamar) from Oxnard

### CODE REVIEW

**Comment by vflyson** [comment]
Hi @egecue, thanks a lot for the contribution! It seems like one or more files in this branch contain spaces at end of lines, one of the tests fails. If you could you please also remove "VERSE 2" from Venom, that'd make them super-readable. I'll merge the texts in and deploy to the website once the tests are passing. Thank you again.

File: database/L/Little Simz/GREY Area/Venom [hunk 1]
```diff
@@ -0,0 +1,91 @@
+Life sucks and I never tried suicide
+Minds fucked even more than I realise
+Times up keep it moving when she arrives
+If you ever heard what I heard in my mind
+Never try
+You would cry
+Thats a lie 
+You would die
+I don’t ever wanna come down from a high
+Im in the best seat
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: I think it should be "I'm" here

File: database/L/Little Simz/GREY Area/Venom [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+If your coming come at me directly
+Don’t need no-one to defend me 
+Souls in a place even I can’t get to
+Don’t fuck with the deadly
+Moon waves and open seas
+Quick coasting over seas
+Fuck those who don’t believe
+They will never wanna admit I’m the best here
+For the mere fact that I’ve got ovaries
+Its a woman world so to speak
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: Perhaps it's supposed to be "It's" here

File: database/L/Little Simz/GREY Area/Venom [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+Never giving credit where its due cos you don’t like pussy in power
+Venom 
+Venom
+
+My my if you ever heard what I heard in the night
+What a fright
+Must’ve been a parasite in my past life
+I don’t ever wanna come down from a high
+My soul ascended to the sky
+Its just you and I
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: Should likely be "It's" here

File: database/L/Little Simz/GREY Area/Venom [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+Make a move better pattern it quick
+I assume you’ll be coming for blood 
+That makes 2 of us that makes 2 of us
+Venom 
+
+Venom
+Venom
+Venom 
+
+VERSE 2
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: This is not part of the text, if it's not audible in the song it perhaps should be removed

File: database/L/Little Simz/GREY Area/Venom [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+Part of the day in the life you won’t see
+Part of the day in the life you can’t be here for
+Its the day in the life of OG’s
+Dealing with inner demons that won’t leave
+No a word you will listen when my soul speaks
+Down I go
+So follow me follow me follow me
+Actually don’t follow me
+Nobody bother me
+Im a mess honestly
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: Another apostrophe missing

File: database/L/Little Simz/GREY Area/Venom [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+Its the day in the life of OG’s
+Dealing with inner demons that won’t leave
+No a word you will listen when my soul speaks
+Down I go
+So follow me follow me follow me
+Actually don’t follow me
+Nobody bother me
+Im a mess honestly 
+Fuck all your policies
+They wanna kick me down a demolish me
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: Might be "kick me down and demolish me"

File: database/L/Little Simz/GREY Area/Venom [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+Use me and fucking abolish me
+I don’t wanna hear no apologies
+Or am I in my own head?
+Can’t tell anymore psychotic to the death
+When I dig deep I can never find nothing left
+Its a mystery
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: Missing apostrophe

File: database/L/Little Simz/GREY Area/Venom [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+Use me and fucking abolish me
+I don’t wanna hear no apologies
+Or am I in my own head?
+Can’t tell anymore psychotic to the death
+When I dig deep I can never find nothing left
+Its a mystery 
+Rage
+Nothing but rage
+Can’t figure out if I’m going insane 
+Aint’ no doubt we’ll be showing no mercy so think twice if you wanna get blazed
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: Perhaps should be "Ain't" instead

File: database/L/Little Simz/GREY Area/Venom [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+Nothing ain’t nice round here no games
+I don’t wanna dark nobody in a flash
+True its a stick up
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: Likely should be "it's"

File: database/L/Little Simz/GREY Area/Venom [hunk 10]
```diff
@@ -0,0 +1,91 @@
+Life sucks and I never tried suicide
+Minds fucked even more than I realise
+Times up keep it moving when she arrives
+If you ever heard what I heard in my mind
+Never try
+You would cry
+Thats a lie
```
Comment (vflyson) [inline database/L/Little Simz/GREY Area/Venom]: Likely should be "That's"

**Comment by egecue** [comment]
@vflyson I've fixed.

**Comment by egecue** [comment]
Wrong button lol, sorry.

**Comment by vflyson** [comment]
I've added a review, but will merge the code in right now as promised, please feel free to fix it up if you feel like it, I'll likely add more tests to make it easier to spot "verse" bits and missing apostrophes

**Comment by egecue** [comment]
> I've added a review, but will merge the code in right now as promised, please feel free to fix it up if you feel like it, I'll likely add more tests to make it easier to spot "verse" bits and missing apostrophes

**Comment by vflyson** [comment]
Thank you again for the contribution. The lyrics can now be found here https://lyrics.github.io/db/l/little-simz/grey-area/venom/
Feel free to make more PRs, I'll be glad to review and merge them into the database

**Pull request merged**